ext/civi_event/Civi/Api4/Service/Spec/Provider/EventGetSpecProvider.php

```diff
@@ -48,12 +48,26 @@ public function applies($entity, $action) {
    * @param array $maxField
    * @param \Civi\Api4\Query\Api4SelectQuery $query
    * return string
+   *
+   * @throws \CRM_Core_Exception
    */
   public static function getRemainingParticipants(array $maxField, Api4SelectQuery $query): string {
-    $statuses = \CRM_Event_PseudoConstant::participantStatus(NULL, 'is_counted = 1');
+    $statuses = \CRM_Event_BAO_Participant::buildOptions('status_id', NULL, ['is_counted' => 1]);
```

Comment:
This looks good but just to avoid the very edgiest of edge-cases let's avoid a sql error if there are no counted roles:
```suggestion
    $statuses = \CRM_Event_BAO_Participant::buildOptions('status_id', NULL, ['is_counted' => 1]) ?: [0];
```